apps/web/src/components/auth/RegisterForm.tsx

```diff
@@ -17,203 +17,109 @@ import { Form } from 'ui';
 import * as z from 'zod';
 
 import { StyledContainer } from './styles';
-import { useRegister } from '../../lib/auth';
 
-const addressSchema = z.object({
-  street: z.string(),
-  city: z.string(),
-  region: z.string(),
-  zipcode: z.string(),
-  country: z.string(),
+const schema = z.object({
+  email: z.string().min(1, 'Required'),
+  password: z.string().min(8, 'Minimum Password is 8'),
 });
 
-const schema = z
-  .object({
-    userType: z.string(),
-    username: z.string().regex(/^[a-zA-Z0-9_]*$/, {
-      message:
-        'Username must contain only alphanumeric characters and no spaces.',
-    }),
-    firstName: z
-      .string()
-      .min(2, { message: 'First Name must be at least 2 characters long.' }),
-    lastName: z
-      .string()
-      .min(2, { message: 'Last Name must be at least 2 characters long.' }),
-    email: z.string().email({
-      message:
-        'Invalid email format. Please provide a valid email address (example@example.com).',
-    }),
-    address: addressSchema,
-    phoneNumber: z
-      .string()
-      .max(13, { message: 'Phone Number exceeds 10' })
-      .includes('+63', {
-        message: 'Include Country Code (ex. +639123456789)',
-      }),
-    password: z
-      .string()
-      .regex(/^(?=.*[A-Za-z])(?=.*\d)(?=.*[@$!%*#?&])[A-Za-z\d@$!%*#?&]{8,}$/, {
-        message:
-          'Password must be at least 8 characters long and include at least one letter, one digit, and one special character.',
-      }),
-    confirm: z.string(),
-  })
-  .refine((data) => data.password === data.confirm, {
-    message: 'Password doesnt match',
-    path: ['confirm'],
-  });
-
-type TRegisterAddress = {
-  street: string;
-  city: string;
-  region: string;
-  zipcode: string;
-  country: string;
-};
-
 type TRegisterValues = {
-  userType: string;
+  usertype: string;
   username: string;
-  firstName: string;
-  lastName: string;
+  firstname: string;
+  lastname: string;
   email: string;
-  address: TRegisterAddress;
-  phoneNumber: string;
+  street: string;
+  region: string;
+  zipcode: number;
+  country: string;
   password: string;
-  confirm: string;
+  confirmpassword: string;
 };
 
 type IRegisterFormProps = {
   onSuccess: () => void;
 };
 
 export const RegisterForm = ({ onSuccess }: IRegisterFormProps) => {
-  const register = useRegister();
-  const [data] = useState([
+  const [data, setData] = useState([
     { value: 'consumer', label: 'Consumer' },
     { value: 'merchant', label: 'Merchant' },
   ]);
 
```

Comment:
```suggestion
const [inputValues, setInputValues] = useState({
    address: '',
    region: '',
    // Add more fields here
  });

  const handleInputChange = (event, fieldName) => {
    const { value } = event.target;
    setInputValues((prevValues) => ({
      ...prevValues,
      [fieldName]: value,
    }));
  };

  const handleInputBlur = (fieldName) => {
    setInputValues((prevValues) => {
      const newValue = prevValues[fieldName].trim();
      return {
        ...prevValues,
        [fieldName]: newValue,
      };
    });
  };
```

add this to trim spaces before and after certain TextInput fields